README.md

```diff
@@ -1,3 +1,53 @@
-# Backend app
+# Backend Music Library
+
+## Description
+
+I built this project in order to expand my knowledge on Mocha and Javascript, and to put my skills to the test with what I have learnt through my course.
+My music library solves the problem of having a neat and convenient way to store your favourite artists, and it allows you and others to see more about that artsit and the albums they have released.
+From this project I have learnt how to use Postman and the API's to add artists and albums. I have also learnt more about express, and the syntax that comes with it.
+
+The dev dependencies and depenencies used in this project are:
+"chai": "^4.3.7",
+"dotenv": "^16.0.3",
+"eslint": "^8.36.0",
+"mocha": "^10.2.0",
+"nodemon": "^2.0.21",
+"supertest": "^6.3.3"
+
+"express": "^4.18.2",
+"pg": "^8.10.0",
+"postgres-migrations": "^5.3.0",
+"prettier": "^2.8.8"
```

Comment:
These arent showing as a list, slight format change:
```
- chai
- dotenv
- eslint
- mocha
- nodemon
- supertest
- express
- pg
- postgres-migrations
- prettier
```
Also you dont need to specify which versions of each of the packages you are using as if people want to know that they can just look at the `package.json` file